nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-processors/src/main/java/org/apache/nifi/processors/kudu/PutKudu.java

```diff
@@ -688,7 +688,16 @@ public OperationType apply(final Record record) {
 
             final String resultValue = String.valueOf(resultList.get(0).getValue());
             try {
-                return OperationType.valueOf(resultValue.toUpperCase());
+                switch (resultValue) {
```

Comment:
Please include the operation "r" (read) which is also interpreted as insert. 
Debezium sends a read event for the rows already existed in database when Debezium engine started.
The records inserted to DB after Debezium has been started are captured as "c" (create) events.